src/status_im2/contexts/share/events.cljs

```diff
@@ -0,0 +1,22 @@
+(ns status-im2.contexts.share.events
+  (:require [utils.re-frame :as rf]
+            [status-im2.common.toasts.events :as toasts]
+            [quo2.foundations.colors :as colors]
+            ;;TODO(siddarthkay) : move the components below over to status-im2 ns
+            ;; issue -> https://github.com/status-im/status-mobile/issues/15549
+            [status-im.ui.components.react :as react]))
+
+(rf/defn open-profile-share-view
+  {:events [:share/open]}
+  [{:keys [db]}]
+  {:dispatch [:open-modal :profile-share {}]})
+
+(rf/defn copy-text-and-show-toast
+  {:events [:share/copy-text-and-show-toast]}
+  [cofx text-to-copy post-copy-message]
+  (react/copy-to-clipboard text-to-copy)
```

Comment:
this shouldn't an fx, could you pls add a comment if you won't implement it in this PR

Reply:
> this shouldn't an fx

I don't follow what you're saying, could you please elaborate?

Reply:
idk, why you have outdated comment, i mean it should be a map  of effects
`{:copy-to-clipboard text-to-copy}`